tests/performingApiRequest.spec.ts

```diff
@@ -0,0 +1,127 @@
+import { test, expect } from '@playwright/test';
+import { PageManager } from '../pages/pageManager'
+
+
+test.beforeEach( async({page}) => {
+    await page.goto('/')
+});
+
+test('Delete Specialty', async ({page, request}) =>{
+    const specialtyName = "api testing expert"
+    const specialtiesResponse = await request.post('https://petclinic-api.bondaracademy.com/petclinic/api/specialties', {
+        data: {
+            name: specialtyName
+        }
+    })
+    expect(specialtiesResponse.status()).toEqual(201)
+      
+    const pm = new PageManager(page)
+
+    await pm.navigateTo().specialtiesPage()
+
+    await pm.onSpecialtiesPage().validateSpecialtyExistsFor(specialtyName)
+    
+    await pm.onSpecialtiesPage().deleteSpecialty(specialtyName)
+    
+})
+
+test('Add and Delete Veterinarian', async ({page, request}) => {
+    const firstName = 'Paula'
+    const lastName = 'Hutchinson'
+    const specialty = 'dentistry'
+
+    // Create a new veterinarian
+    const veterinarianCreationResponse = await request.post('https://petclinic-api.bondaracademy.com/petclinic/api/vets',{
+        data: {
+            'id': null,
+            'firstName': firstName,
+            'lastName': lastName,
+            'specialties': []
+        }
+    })
+
+    // Select and validate specialty for new Vet
+    const vetID = (await veterinarianCreationResponse.json()).id
+    
+    const pm = new PageManager(page)
+
+    await pm.navigateTo().veterinariansPage()
+
+    await pm.onVeterinariansPage().validateSpecialtyCountFor(lastName, 0)
+
+    await pm.onVeterinariansPage().clickEditVeterinarianFor(lastName)
+
+    await pm.onEditVeterinarianPage().selectSpecialty(specialty)
+
+    await pm.onEditVeterinarianPage().clickSaveVet()
+
+    await pm.onVeterinariansPage().validateSpecialtyFor(lastName, specialty)
+
+    // Delete the veterinarian by API
+    const deleteResponse = await request.delete(`https://petclinic-api.bondaracademy.com/petclinic/api/vets/${vetID}`)
+    expect(deleteResponse.status()).toEqual(204)
+
+    // Get a list of veterinarians by API and validate that our test vet isn't among them
+    const getVeterinariansResponse = await request.get('https://petclinic-api.bondaracademy.com/petclinic/api/vets')
+    const responseBody = await getVeterinariansResponse.json()
+    const testVeterinarian = await responseBody.find(vet => vet.firstName === firstName)
+    expect(testVeterinarian).toEqual(undefined)
+
+})
+
+test('New Specialty is Displayed', async ({page, request}) => {
+    const firstName = 'Leroy'
+    const lastName = 'Jenkins'
+    const newSpecialtyName = "api testing ninja"
+    const oldSpecialtyJSON = {
+        'id': 156, 
+        'name': 'surgery'
+    }
```

Comment:
Instead of hardcoding the ID for the specialty, which will be different for my test account, compared to yours, you can make a get request to get the list of specialties, and then take the specialty you are looking for with a corresponding ID. It will be a cleaner approach